CODEOWNERS

```diff
@@ -0,0 +1,52 @@
+                                                              # Codeowners for the project
+
+
+
+# Global owner for everything in this repo
+* @rt4914
+
+# Owners/teams for backend (Ruby on Rails)
```

Comment:
Remove everything below this.

Reply:
I have also done this and assigned it to you as well

Reply:
@KvngJamesNG You can get rid of last line too. We don't need that comment anymore.